docs/src/main/resources/microsite/css/custom.css

```diff
@@ -0,0 +1 @@
kground:#EFF2F3;color:#104051}#site-footer a{color:#899fa7;transition:color 250ms ease-in-out}#site-footer a:visited{color:#899fa7}#site-footer a:hover{color:#617a82;text-decoration:underline}#site-footer a:active{color:#899fa7}#site-footer p{color:#134a5e}#site-footer .footer-flex{display:-webkit-box;display:-moz-box;display:-ms-flexbox;display:-webkit-flex;display:flex;-webkit-justify-content:space-between;-moz-justify-content:space-between;-ms-justify-content:space-between;justify-content:space-between;-ms-flex-pack:space-between}#site-footer .footer-flex .footer-dev{width:41.6666666667%}#site-footer .footer-flex .footer-dev a{color:#899fa7}#site-footer .footer-flex .footer-dev a:visited{color:#899fa7}#site-footer .footer-flex .footer-dev a:hover{color:#617a82}#site-footer .footer-flex .footer-dev a:active{color:#899fa7}#site-footer .footer-flex .footer-menu{display:-webkit-box;display:-moz-box;display:-ms-flexbox;display:-webkit-flex;display:flex}#site-footer .footer-flex .footer-menu li:not(:last-child){margin-right:32px}@media (max-width: 768px){#site-footer .footer-flex{-webkit-justify-content:center;-moz-justify-content:center;-ms-justify-content:center;justify-content:center;-ms-flex-pack:center;flex-wrap:wrap}#site-footer .footer-flex .footer-dev,#site-footer .footer-flex .footer-menu{width:66.6666666667%}#site-footer .footer-flex .footer-dev{padding-bottom:16px;margin-bottom:16px;text-align:center;border-bottom:1px solid #CED8DC}#site-footer .footer-flex .footer-menu{-webkit-justify-content:center;-moz-justify-content:center;-ms-justify-content:center;justify-content:center;-ms-flex-pack:center}#site-footer .footer-flex .footer-menu li:not(:last-child){margin-right:16px}}#sidebar-wrapper{background-color:#FFF;border-right:1px solid #CED8DC}.sidebar-nav{background-color:#FFF}.sidebar-nav>li>a{color:#104051}.sidebar-nav>li>a.active{background:#EFF2F3;border-left:3px solid #DD4949;color:#104051}.sidebar-nav>li>a:hover{background:#EFF2F3;color:#104051}.sidebar-nav>li>a>span{display:none}.sidebar-nav>li ul li a{background:#EFF2F3;color:#617a82}.sidebar-nav>li ul li a:hover,.sidebar-nav>li ul li a:active{color:#104051}.sidebar-nav>.sidebar-brand a{background-color:#FFF;border-bottom:1px solid #CED8DC;color:#104051;text-transform:uppercase;height:107px}.sidebar-nav>.sidebar-brand a .brand-wrapper{height:60px;width:60px}.sidebar-nav>.sidebar-brand a .brand-wrapper span {margin-top:18px;letter-spacing:4px;font-size:16px;padding-left:32px}#page-content-wrapper{background:#FFF}#page-content-wrapper section pre .hljs{background-color:#EFF2F3}body{color:#104051;font-family:"Poppins",sans-serif}#fetch-home{background:#FFF}ol,ul{list-style:none}.btn{padding:12px 40px;border-radius:0}.btn:hover{background:#104051}.wrapper{padding:0 24px;margin:0 auto;box-sizing:border-box;max-width:1040px}.button{display:block;background:none;border:none;outline:none;text-decoration:none;position:relative}.button:hover{cursor:pointer}.button>img{vertical-align:bottom}
```

Comment:
Would it be possible to set Jekyll to build the CSS instead of leaving the developers bearing that burden?

Reply:
☹